src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -2,6 +2,7 @@
   (:require
     [dmohs.react :as react]
     [clojure.set :refer [union]]
+    [goog.net.cookies :as cks]
```

Comment:
It doesn't look like you're using the short name, so could could just replace:
```clojure
[goog.net.cookies :as cks]
```
with
```clojure
goog.net.cookies
```